fiftyone/utils/sam3.py

```
@@ -0,0 +1,537 @@
+            return self._output_processor(
+                output,
+                orig_image_sizes,
+                confidence_thresh=self.config.confidence_thresh,
+                box_prompts=boxes_xyxy,
+                labels=labels,
+                mask_index=self.config.points_mask_index,
+                classes=self.config.filter_classes,
+            )
+        return output
+
+    def _forward_pass_concept(self, imgs):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_forward_pass_concept`

(ANN202)